source/jormungandr/jormungandr/interfaces/v1/converters_collection_type.py

```diff
@@ -38,7 +38,7 @@
     "pois": "poi", "poi_types": "poi_type",
     "connections": "connection", "vehicle_journeys": "vehicle_journey",
     "disruptions": "disruption", "trips": "trip", "contributors": "contributor",
-    "frames": "frame"}
+    "datasets": "frame"}
```

Comment:
while using ptref like /datasets/id, kraken has to receive filter 'frame.uri=id'.